### PR TITLE
Cast span_id and trace_id as string when adding to the record.

### PR DESCRIPTION
**Pull request by arthurio**
If you are using a json logger the record value will be interpreted as a number which will result in values being rounded in the log pipeline.

### CODE REVIEW

**Comment by majorgreys** [comment]
@arthurio This fix makes good sense and we can certainly use strings for the record attributes. Do you mind sharing an example log entry where this rounding happened? I'm wondering if its a case of an int being truncated rather than rounded.

File: ddtrace/contrib/logging/patch.py [hunk 1]
```diff
@@ -44,8 +44,8 @@ def _w_makeRecord(func, instance, args, kwargs):
         span = _get_current_span(tracer=ddtrace.config.logging.tracer)
 
     if span:
-        setattr(record, RECORD_ATTR_TRACE_ID, span.trace_id)
-        setattr(record, RECORD_ATTR_SPAN_ID, span.span_id)
+        setattr(record, RECORD_ATTR_TRACE_ID, "{}".format(span.trace_id))
+        setattr(record, RECORD_ATTR_SPAN_ID, "{}".format(span.span_id))
```
Comment (majorgreys) [inline ddtrace/contrib/logging/patch.py:48]:
```suggestion
        setattr(record, RECORD_ATTR_TRACE_ID, str(span.trace_id))
        setattr(record, RECORD_ATTR_SPAN_ID, str(span.span_id))
```

**Comment by arthurio** [comment]
>This fix makes good sense and we can certainly use strings for the record attributes. Do you mind sharing an example log entry where this rounding happened? I'm wondering if its a case of an int being truncated rather than rounded.

@majorgreys Thanks for getting back to me, this is the screenshot I took when reporting the bug:

Live_Tail___Datadog_and_Forum___Minerva_Project

FYI we use `python-json-logger`.

**Comment by jd** [comment]
It'd be great to have a test so we don't break it in the future.

File: tests/contrib/logging/test_logging.py [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+from ddtrace.constants import ENV_KEY, VERSION_KEY
 from ddtrace.contrib.logging import patch, unpatch
+from ddtrace.contrib.logging.patch import RECORD_ATTR_TRACE_ID, RECORD_ATTR_SPAN_ID
 from ddtrace.vendor import wrapt
 
 from ... import TracerTestCase
 
+PY2 = sys.version_info[0] == 2
+
+if PY2:
```
Comment (Kyle-Verhoog) [inline tests/contrib/logging/test_logging.py:15]: We can use `six.string_types` here available from `ddtrace.vendor.six` 🙂

Comment (arthurio) [reply tests/contrib/logging/test_logging.py]: Thanks for the pointer, I was trying to use code I found in other parts of the codebase 😛

**Review by Kyle-Verhoog — DISMISSED** [review]
Thanks for the test and quick response @arthurio!! 😄

**Comment by arthurio** [comment]
@Kyle-Verhoog No problem, thanks for your help. And are those assertions enough testing?

**Comment by Kyle-Verhoog** [comment]
Yeah I think it'd be overkill to do anything further. Future people can reference this PR for the justification.

**Comment by Kyle-Verhoog** [comment]
Awh :/ looks like Python 2 doesn't like the filter approach: https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/2409/workflows/b68ce846-7572-42a1-8b2c-126a6f2745cb/jobs/322063

**Comment by arthurio** [comment]
@Kyle-Verhoog Ok, I switched to a filter class to accommodate for both.

**Pull request merged**